packages/opal-server/opal_server/policy/watcher/factory.py

```diff
@@ -27,6 +27,7 @@ def setup_watcher_task(
     policy_bundle_token: str = None,
     policy_bundle_token_id: str = None,
     policy_bundle_server_type: str = None,
+    policy_bundle_aws_region: str = None,
```

Comment:
That's not crucial. I think simply passing `region=opal_server_config.POLICY_BUNDLE_SERVER_AWS_REGION` to `ApiPolicySource` is enough

Reply:
Apologies,  this was like 10 projects ago; could you please elaborate on this?

Reply:
I mean it's probably not really needed to have `policy_bundle_aws_region` as a keyword argument to `setup_watcher_task` (since the caller doesn't set it, and you load it from configuration here anyway).

But never mind - let's move on :)